src/Akka.Persistence.SqlServer.Tests/Akka.Persistence.SqlServer.Tests.csproj

```diff
@@ -1,12 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-        <TargetFrameworks>$(NetFrameworkTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
-    </PropertyGroup>
-
-    <!-- disable .NET Framework (Mono) on Linux-->
-    <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
-        <TargetFrameworks>$(NetCoreTestVersion)</TargetFrameworks>
+    <PropertyGroup>
+        <TargetFramework>$(NetCoreTestVersion)</TargetFramework>
```

Comment:
The `IAsyncEnumerable` stuff has never been tested on .NET 4.6.1, according to Microsoft. Dropped .NET Framework as a test target. Can add it back if people really want it.